dedoc/api/api_args.py

```diff
@@ -24,7 +24,7 @@ class QueryParameters:
     table_type: str = Form("", description="Pipeline mode for table recognition")
 
     # pdf handling
-    pdf_with_text_layer: str = Form("auto_tabby", enum=["true", "false", "auto", "auto_tabby", "tabby"],
+    pdf_with_text_layer: str = Form("auto_tabby", enum=["true", "false", "auto", "auto_tabby", "tabby","bad_encoding_reader"],
```

Comment:
API test should be written for a new parameter: you can create file `test_pdf_bad_encoding_reader.py` similarly to [test_api_format_pdf_with_text.py](https://github.com/ispras/dedoc/blob/master/tests/api_tests/test_api_format_pdf_with_text.py)